skytemple_randomizer/randomizer/boss.py

```diff
@@ -222,10 +228,47 @@ def run(self, status: Status):
                 for bi in ACTOR_TO_BOSS_MAPPING[i]:
                     boss_list[bi].md_idx = actor.entid
 
+        status.step(_("Updating bazaar monsters..."))
```

Comment:
I've elected to split this into its own step from above, as bazaar monsters aren't "bosses" and therefore would be better suited to be a step of its own for clarity.